### PR TITLE
update LIBLINEAR from 2.1 to 2.11

### PR DESCRIPTION
**Pull request by wvengen**
Small upstream changes, but possibly useful nonetheless.
Specs succeed, specs for a real-world project using this also work, but not tested in production.
Sorry about the whitespace changes, upstream [also has](https://github.com/cjlin1/liblinear/compare/v201...v211) whitespace changes.

It occurs to me that the liblinear-2.11 directory could be completely removed, right?

### CODE REVIEW

File: lib/liblinear.rb [hunk 1]
```diff
@@ -26,6 +26,8 @@ class Liblinear
   L2R_L2LOSS_SVR_DUAL = Liblinearswig::L2R_L2LOSS_SVR_DUAL
   L2R_L1LOSS_SVR_DUAL = Liblinearswig::L2R_L1LOSS_SVR_DUAL
 
+  LIBLINEAR_VERSION   = sprintf("%.2f", Liblinearswig::LIBLINEAR_VERSION / 100.0)
+
```
Comment (wvengen) [inline lib/liblinear.rb]: Why not the integer version number? `Liblinear::VERSION` is also a string, and this is forward-compatible with non `x.yy` version numbers if they would be occuring in the future.

Comment (kei500) [inline lib/liblinear.rb]: 👍

**Comment by kei500** [comment]
@wvengen 

Thank you very much for your PR.
I've also confirmed that all tests are passed.

> It occurs to me that the liblinear-2.11 directory could be completely removed, right?

You are right.
This directory is unnecessary.
After I merge this PR, I'll delete this directory.

**Comment by kei500** [comment]
v1.0.2 is released!
https://rubygems.org/gems/liblinear-ruby/versions/1.0.2

**Comment by wvengen** [comment]
Thanks for your quick integration!

**Pull request merged**